packages/api/tests/endpoints/providers/update-provider.js

```diff
@@ -60,14 +80,24 @@ test.beforeEach(async (t) => {
     ...fakeProviderFactory(),
     cmKeyId: 'key',
   };
-  await providerModel.create(t.context.testProvider);
+  await request(app)
+    .post('/providers').send(t.context.testProvider)
```

Comment:
We have `.send` set on this line and the one below

Reply:
👍  good catch 66d111a59